cloud_blobstore/__init__.py

```diff
@@ -249,6 +249,16 @@ class BlobStoreCredentialError(BlobStoreError):
     pass
 
 
+class BlobStoreTimeoutError(BlobStoreError):
+    """
+    BlobStoreTimeoutError wraps timeout erros from boto3, which arrise during difficulty when making
```

Comment:
Not just boto3.  Presumably this applies to any cloud provider.

Reply:
Generalized doc string.